src/admin/adminActions.ts

```diff
@@ -129,6 +129,57 @@ export const awardPoints: ThunkActionCreator = (pointDetails: any) => async (dis
   });
 };
 
+export const addAttendance: ThunkActionCreator = (attendanceDetails: any) => async (dispatch) => {
+  return new Promise(async (resolve, reject) => {
+    if (!attendanceDetails.event) {
+      notify('Validation Error!', 'No event specified');
+      reject();
+      return;
+    }
+    if (!attendanceDetails.attendees || attendanceDetails.attendees.length === 0) {
+      notify('Validation Error!', 'No attendees added');
+      reject();
+      return;
+    }
+    try {
+      const url = `${Config.API_URL}${Config.routes.admin.attendance}`;
+      await fetchService(url, 'POST', 'json', {
+        requiresAuthorization: true,
+        payload: JSON.stringify({
+          users: attendanceDetails.attendees,
+          event: attendanceDetails.event,
+          asStaff: true,
```

Comment:
Is `asStaff` supposed to be `true` here? I'm pretty sure that's only used for when someone checks into an event as a staff member, which wouldn't apply in this use case.

Reply:
Oh, I thought it was to check if the person submitting the retro add was staff! I'll fix it and add a checkmark for staff adding